fda-model/structures/Inventory.cs

```diff
@@ -57,10 +57,6 @@ public static T TryGetObj<T>(object value, T defaultValue = default)
         }
 
 
-        /// <summary>
-        /// Constructor to create a SI from a shapefile. Gonna need to do this from database potentially as well
-        /// </summary>
-        /// <param name="pointShapefilePath"></param>
         public Inventory(string pointShapefilePath, string impactAreaShapefilePath, StructureInventoryColumnMap map)
```

Comment:
I think we're going to need to add List<OccupancyType> to this constructor. I am going to look at it today.